packages/blade/src/components/SideNav/tokens.ts

```diff
@@ -15,7 +15,8 @@ const classes = {
 const SKIP_NAV_ID = 'blade-side-nav-skip';
 
 const COLLAPSED_L1_WIDTH = size['56'];
-const EXPANDED_L1_WIDTH = size['264'];
+const EXPANDED_L1_WIDTH_DESKTOP = size['264'];
+const EXPANDED_L1_WIDTH_MOBILE = size['245'];
```

Comment:
why do we have width on mobile? width on mobile is just width of Drawer no?

Reply:
Just realised its width on smaller desktops. Can you rename it here?

Reply:
> Can you rename it here?

Suggest a variable name.

Reply:
I meant something like EXPANDED_L1_WIDTH_XL, EXPANDED_L1_WIDTH_BASE

or have it as JSON EXPANDED_L1_WIDTH = { base: '', xl: '' } 

The "mobile" is bit confusing because its not for mobile

Reply:
Makes sense, EXPANDED_L1_WIDTH_XL.